[irobot] Failing test

Build on Windows:
```
[INFO] -------------------------------------------------------
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running org.openhab.binding.irobot.internal.handler.RoombaHandlerTest
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.016 s <<< FAILURE! - in org.openhab.binding.irobot.internal.handler.RoombaHandlerTest
[ERROR] org.openhab.binding.irobot.internal.handler.RoombaHandlerTest  Time elapsed: 0.016 s  <<< ERROR!
java.lang.IllegalArgumentException: Last segment must not be blank.
        at org.openhab.binding.irobot.internal.handler.RoombaHandlerTest.<init>(RoombaHandlerTest.java:63)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   RoombaHandlerTest.<init>:63 ▒ IllegalArgument Last segment must not be blank.
[INFO]
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
```

--- a/bundles/org.openhab.binding.irobot/src/test/java/org/openhab/binding/irobot/internal/handler/RoombaHandlerTest.java
+++ b/bundles/org.openhab.binding.irobot/src/test/java/org/openhab/binding/irobot/internal/handler/RoombaHandlerTest.java
@@ -60,7 +60,7 @@ public class RoombaHandlerTest {
     @Nullable
     private RoombaHandler handler;
     // We have to initialize it to avoid compile errors
-    private @Mock Thing thing = new ThingImpl(new ThingTypeUID("AA:BB"), "");
+    private @Mock Thing thing = new ThingImpl(new ThingTypeUID("AA:BB"), "test");
     @Nullable
     private ThingHandlerCallback callback;
 

